app/src/main/java/pm/gnosis/heimdall/ui/qrscan/Videographer.kt

```diff
@@ -43,7 +48,7 @@ class Videographer(val activity: Activity) {
         }
     }
 
-    lateinit var onSuccessfulScan: (result: String) -> Unit
+    var onSuccessfulScan: (result: String) -> Unit = {}
```

Comment:
I am always more a fan of nullable callbacks, to avoid unnecessary creation of empty function classes